packages/wrangler/src/index.tsx

```diff
@@ -1924,13 +1864,15 @@ export async function main(argv: string[]): Promise<void> {
                   throw new Error("No account id found, quitting...");
```

Comment:
could you add the snip comment back here too please

Reply:
Done